src/components/Modeler.vue

```diff
@@ -180,6 +190,11 @@ export default {
     autoValidate() { this.validateIfAutoValidateIsOn(); },
   },
   computed: {
+    tooltipTitle() {
+      if (this.tooltipTarget) {
+        return this.tooltipTarget.$el.data('title');
+      }
```

Comment:
@tuantruong625  @chipit24  the only thing bugging me about this is what will happen if there is no title? Will it show a tool tip with the text "undefined". 

I'm thinking of a scenario where a new component gets added which doesn't have the title set in crown config. Can there be a safe default, like if no title dont show anything?

Reply:
`Crown Button` as a fall back?

<img width="296" alt="Screen Shot 2019-06-19 at 4 50 31 PM" src="https://user-images.githubusercontent.com/26545455/59799462-6225f700-92b2-11e9-8e55-03d307b79191.png">

Reply:
@ssshake @velkymx Will create a separate issue for how to handle if title is undefined.